joomla/joomla-cms
# Messages inside the new Joomla 3.6 updater / max post size 8mb

Pull Request for Issue #10604 
#### Summary of Changes

change extensions to packages
#### Testing Instructions
- set maximum post size to less than 8MB
- see the message in the updater
- see the message in the extension manger

similiar to https://github.com/joomla/joomla-cms/pull/10707 Thanks @brianteeman
